utils/training_loop.py

```diff
@@ -59,14 +64,17 @@ def plot_results(experiment, all_rewards, all_actions):
 
 
 def run_experiment(experiment):
-    logger = configure(experiment.EXPERIMENT_PATH, ["stdout", "csv", "tensorboard"])
     airsim_manager = AirsimManager(experiment)
     env = DummyVecEnv([lambda: Agent(experiment, airsim_manager)])
     agent = Agent(experiment, airsim_manager)
-    model = PPO(policy=PPO_MLP_Policy, env=env, verbose=1,
-                learning_rate=experiment.LEARNING_RATE,
-                n_steps=experiment.N_STEPS,
-                batch_size=experiment.BATCH_SIZE)
+    # Initialize model selector with the desired type and configuration
+    model_selector = ModelSelector(experiment.MODEL_TYPE, env, config={
+        'learning_rate': experiment.LEARNING_RATE,
+        'n_steps': experiment.N_STEPS,
+        'batch_size': experiment.BATCH_SIZE
+    })
+    model = model_selector.get_model()
+    logger = configure(experiment.EXPERIMENT_PATH, ["stdout", "csv", "tensorboard"])
     model.set_logger(logger)
 
     model, collision_counter, all_rewards, all_actions = training_loop(experiment=experiment, env=env, agent=agent,
```

Comment:
In the line of 
def resume_experiment_simulation(env):
Complete the TODO (why not using resume_simulation from airsim manager?)

Reply:
Done